x-pack/plugins/security_solution/public/common/components/exceptions/builder/builder_button_options.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiButton } from '@elastic/eui';
+import styled from 'styled-components';
+
+import * as i18n from '../translations';
+
+const MyEuiButton = styled(EuiButton)`
+  min-width: 95px;
+`;
+
+interface BuilderButtonOptionsProps {
+  isOrDisabled: boolean;
+  isAndDisabled: boolean;
+  displayInitButton: boolean;
+  showNestedButton: boolean;
+  onAndClicked: () => void;
+  onOrClicked: () => void;
+  onNestedClicked: () => void;
+}
+
+export const BuilderButtonOptions: React.FC<BuilderButtonOptionsProps> = ({
+  isOrDisabled = false,
+  isAndDisabled = false,
+  displayInitButton,
+  showNestedButton = false,
+  onAndClicked,
+  onOrClicked,
+  onNestedClicked,
+}) => (
+  <EuiFlexGroup gutterSize="s" alignItems="center">
+    {displayInitButton ? (
+      <EuiFlexItem grow={false}>
+        <EuiButton
+          fill
+          size="s"
+          iconType="plusInCircle"
+          onClick={onOrClicked}
```

Comment:
Not an error here that it says `onOrClicked` as they both do the same thing - create a new exception list item.